src/build.py

```diff
@@ -749,6 +749,12 @@ def GetRepoInfo():
 
 # Build rules
 
+def RunToolTests():
+  """For local debugging it can be useful to build tools without running
+  internal tests first."""
+  return 'SKIP_TESTS' not in os.environ
```

Comment:
Thus far we have managed to avoid dependence on the environment for anything other than buildbot-specific stuff. 

Actually, thinking a little bit bigger, this is now the third "option" that modifies the build steps The others being disabling for certain OSes, and the `use_asm` option for emscripten, (which essentially has the same purpose as this one; i.e. disabling some build work when you just want to do a fast local build). We talked previously about making a more generic mechanism for the OS matrix; maybe it's time for a more general way for the build steps to take options. Not sure how ugly threading everything through the `Build` constructor would be, maybe there's even a better way. Or maybe it is just cleaner to do some global thing, I don't know. But we should probably at least ask the question.